aisikl/components/radiobox.py

```diff
@@ -60,6 +61,13 @@ def changed_properties(self):
     def _ais_setDataView(self, id, body):
         data_view = body.find(id=id)
         container = data_view.contents[0]
+
+        # If we found a NavigableString, something is probably wrong
+        # and bailing by trying to find radioBox_container seems like
+        # the right thing to do
+        if type(container) == NavigableString:
```

Comment:
@TomiBelan takto som sa snazil fixnut chyby typu

```
  File "/eprihlaska/venv/lib/python3.5/site-packages/bs4/element.py", line 737, in __getattr__
    self.__class__.__name__, attr))

AttributeError: 'NavigableString' object has no attribute 'get'
```

Neviem, ci je to moc naivne, ale parsovanie AISoveho responsu mi uz potom fungovalo (aspon v tom malom priklade, ktory riesim).